database-emulator/javascript-quickstart/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
1) You don't need a `LICENSE` within the folder, there's one at the repo root.
2) If you want one, it should be Apache 2.0
3) 2018